src/main/java/spring/controllers/UserController.java

```diff
@@ -0,0 +1,68 @@
+package spring.controllers;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+import spring.dto.UserResponseDto;
+import spring.model.User;
+import spring.service.UserService;
+
+@RequestMapping("/user")
+@RestController
+public class UserController {
+    private UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @ResponseBody
+    @GetMapping("/all")
```

Comment:
according to homework description the corresponding url for getAll():  /user/

Reply:
https://mate-academy.github.io/jv-program-fulltime/04_spring/checklist/hw_32.html

Reply:
done